examples/js/animation/MMDAnimationHelper.js

```diff
@@ -591,6 +592,24 @@ THREE.MMDAnimationHelper = ( function () {
 
 		},
 
+		// Record original bone on userData
+		// because position grant need this.
+		_setupBoneOriginal: function ( mesh ) {
+
+			var bones = mesh.skeleton.bones;
+
+			for( var i = 0; i < bones.length; i++ ) {
+			
+				var bone = bones[i];
+
+				bone.userData = bone.userData || {};
+				bone.userData.MMD = bone.userData.MMD || {};
+				bone.userData.MMD.originalPos = bone.position.clone();
```

Comment:
`MMDAnimationHelper` attempts not to pollute Three.js objects even their `userData`. It saves data/objects it creates in `this.objects` instead. Please consider the same way.